lib/tapioca/compilers/dsl/active_record_fixtures.rb

```diff
@@ -0,0 +1,87 @@
+# typed: strict
+# frozen_string_literal: true
+
+begin
+  require "rails"
+  require "active_record"
+  require "active_record/fixtures"
+  require "active_support/test_case"
+rescue LoadError
+  return
+end
+
+module Tapioca
+  module Compilers
+    module Dsl
+      # `Tapioca::Compilers::Dsl::ActiveRecordFixtures` decorates RBIs for test fixture methods
+      # that are created dynamically by Rails.
+      #
+      # For example, given an application with a posts table, we can have a fixture file
+      #
+      # ~~~yaml
+      # first_post:
+      #   author: John
+      #   title: My post
+      # ~~~
+      #
+      # Rails will allow us to invoke `posts(:first_post)` in tests to get the fixture record.
+      # The generated RBI by this generator will produce the following
+      #
+      # ~~~rbi
+      # # test_case.rbi
+      # # typed: true
+      # class ActiveSupport::TestCase
+      #   sig { params(fixture_names: Symbol).returns(Post) }
+      #   def posts(*fixture_names); end
+      # end
+      # ~~~
+      class ActiveRecordFixtures < Base
+        extend T::Sig
+
+        class FixtureLoader
+          include ActiveRecord::TestFixtures
+        end
+
+        sig { override.params(root: RBI::Tree, constant: T.class_of(ActiveSupport::TestCase)).void }
+        def decorate(root, constant)
+          FixtureLoader.fixture_path = Rails.root.join("test", "fixtures")
```

Comment:
As far as I know, this path is configurable, so should we try to understand what the configuration is?